src/tests/AttributesManagersTest.cpp

```diff
@@ -16,7 +16,6 @@
 
 #include "configure.h"
 
-namespace {
```

Comment:
I've been trying to use a named namespace "Test" (to match some of the existing corrade tests).  I think I just missed that one.

Reply:
In Magnum itself I have `namespace Test { namespace {`, so both :) 

To align with naming conventions here, i'd use lowercase `test` tho. And I use the namespace only to avoid test classes / helpers to get suggested by IDE autocompletion. Not sure if that's important here, I think it could do entirely without the `test` namespace as well.

Reply:
so just annonymous?